NotificationService/NotificationService/Constants/SettingConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NotificationService.Constants;
+public static class SettingConstants
```

Comment:
Move to appsettings?

Reply:
This is part of console app, so there is no appsetting file

Reply:
you can add